app/views/users/new.html.erb

```diff
@@ -14,10 +14,12 @@
     <div class='password-field info'>
       <%= form.password_field :password, placeholder: "Password..." %>
     </div>
-
-    <div class='text-field'>
-      <%= form.label :avatar %>
-      <%= form.file_field :avatar %>
+    <div class="choose-file">
+    <input id="uploadFile" placeholder="Avatar File" disabled="disabled" />
```

Comment:
I think that input tag is there just to show user feedback, and it's value is changed to the name of whatever file the user selects.